src/commands/bot-owner/blacklist.ts

```diff
@@ -20,7 +20,7 @@ export default class BlacklistCommand extends Command {
 
 	public constructor(store: CommandStore, file: string[], directory: string) {
 		super(store, file, directory, {
-			description: (language: any) => language.get('COMMAND_BLACKLIST_DESCRIPTION'),
```

Comment:
Why was this changed?

Reply:
This was not actually being called and was not returning a proper description.
![image](https://user-images.githubusercontent.com/6025520/78454702-b6df0680-7667-11ea-824e-38fbe92aa9ca.png)